gnucash/gnome-utils/gnc-autoclear.cpp

```diff
@@ -225,18 +224,17 @@ gnc_autoclear_get_splits (Account *account, gnc_numeric toclear_value,
     }
     else
         /* copy GList because std::unordered_map value will be freed */
-        *splits = g_list_copy (sack[toclear_value.num]);
+    {
+        *splits = nullptr;
+        for (auto& s : sack[toclear_value.num])
+            *splits = g_list_prepend (*splits, s);
+    }
```

Comment:
I don't know which works faster. Sticking with GLists (assuming you need those to interact with the rest of gnucash) and avoid copying using std::move or working with a std::vector internally in autoclear and copying between GLists and std::vectors at the interface boundaries.
Long-term I suspect all split lists will eventually become std:vectors so at that point it will definitely make sense. Depending on your performance tests it may be too early to do so here already.